File: scala/src/main/scala/tennis/game1/EqualScore.scala
package tennis.game1

object EqualScore {

  def getScore(score : Int): String ={
    score match {
      case 0 =>  "Love-All"
      case 1 => "Fifteen-All"
      case 2 => "Thirty-All"
      case _ => "Deuce"
    }
  }
}

File: python/src/TennisGame1/equal_score.py
from python.src.TennisGame1 import Scoring


class EqualScore(Scoring):
    def __init__(self, p1points):
        self.p1points = p1points

    def compute_score(self):
        return {
            0: "Love-All",
            1: "Fifteen-All",
            2: "Thirty-All",
        }.get(self.p1points, "Deuce")